cloudsync/event.py

```diff
@@ -138,6 +139,11 @@ def do(self):
         try:
             self._reconnect_if_needed()
             self._do_unsafe()
+        except CloudRootMissingError as e:
+            log.exception("Root path/oid error -- stopping sync. %s", e)
+            if self.__nmgr:
+                self.__nmgr.notify_from_exception(SourceEnum(self.side), e)
+            self.stop()
```

Comment:
no, we should not stop the event manager.  the event manager manages the reconnection and reauthentication needed to recover.  see how the first part is "reconnnect_if_needed" ... well this error represents a "reconnect needed" situation.

Reply:
made the adjustment - thread keeps going but event processing is skipped in the invalid root scenario